cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -106,7 +106,9 @@ def setUp(self):
         super().setUp()
         self.fullcourse = CourseFactory.create()
         self.course_setting_url = get_url(self.course.id, 'advanced_settings_handler')
-        self.non_staff_client, _ = self.create_non_staff_authed_user_client()
+
+        self.non_staff_client, self.nonstaff = self.create_non_staff_authed_user_client()
+        CourseStaffRole(self.course.id).add_users(self.nonstaff)
```

Comment:
Can we add a reason why we are doing this?

Reply:
Done.